services/repository-managers/src/test/java/org/sagebionetworks/repo/manager/entity/EntityAuthorizationManagerAutowireTest.java

```diff
@@ -356,7 +375,71 @@ public void testHasAccessDownloadWithMetAccessRestrictions() {
 	}
 
 	@Test
-	public void testHasAccessDownloadWithUnmetAccessRestrictions() {
```

Comment:
missing test where there are multiple ARs on an Entity where one is eligible and the other is not.